scripts/shelltracker/functions/fn_onFired.sqf

```diff
@@ -0,0 +1,58 @@
+/**
+Visualize flying shells for Zeus.
+This should be 100% LOCAL only, as it's PFH!
+*/
+if !(hasInterface) exitWith {};
+params ["_projectile", "_gunner", "_magazine"];
+
+// Drawing shell markers for Zeus and streamers.
+private _markerColor = [side _gunner, true ] call BIS_fnc_sideColor;
+private _ammo = getText(configfile >> "CfgMagazines" >> _magazine >> "ammo");
+private _ammoname = getText(configfile >> "CfgMagazines" >> _magazine >> "displayName");
+private _radius = getNumber(configfile >> "CfgAmmo" >> _ammo >> "indirectHitRange") * 2;
```

Comment:
@senshi-x I understand the logic with the `indirectHitRange` now. Thanks for taking the time to explain it to me. 🙂 

The factor of 2 seems too big though, because the result looks like this:
![image](https://github.com/gruppe-adler/Shoot_and_Scoot.Tanoa/assets/76476468/ddbf25a9-446d-4343-8a8f-4ff78b343e42)
But this area is way too big to realistically cause damage to. See screenshot below for scale of the village and the dirt that is thrown up:
![image](https://github.com/gruppe-adler/Shoot_and_Scoot.Tanoa/assets/76476468/26ebb99f-5453-4123-ba04-ecf490f434e6)

I recommend lowering the factor down to `1.2` where the shell impact has lost only 50% of its damage potential.
![image](https://github.com/gruppe-adler/Shoot_and_Scoot.Tanoa/assets/76476468/8fc3ac92-5f39-4653-8b1f-3beb70f6cfb2)
(see https://community.bistudio.com/wiki/Arma_3:_Damage_Description#IndirectHit)

Reply:
Changed as requested in 2fa6dfe611d133c91cd31115529f8be0cab5f9c5